docs/examples/lib/2-d2oracle/d2oracle.go

```diff
@@ -17,9 +16,8 @@ func main() {
 	// From one.go
 	ruler, _ := textmeasure.NewRuler()
 	_, graph, _ := d2lib.Compile(context.Background(), "x -> y", &d2lib.CompileOptions{
-		Layout:  d2dagrelayout.DefaultLayout,
-		Ruler:   ruler,
-		ThemeID: d2themescatalog.GrapeSoda.ID,
```

Comment:
why was this removed?

Reply:
The `Compile` method is now independent of the theme colors. It uses theme placeholder values like `N1`, `N2`, `B2` defined in `lib/color/color.go`.